python/pyspark/mllib/stat/_statistics.py

```diff
@@ -16,51 +16,57 @@
 #
 
 import sys
+from typing import overload, List, Optional, Union
+from typing_extensions import Literal
+
+from numpy import ndarray
 
 from pyspark.rdd import RDD
 from pyspark.mllib.common import callMLlibFunc, JavaModelWrapper
-from pyspark.mllib.linalg import Matrix, _convert_to_vector
+from pyspark.mllib.linalg import Matrix, Vector, _convert_to_vector  # type: ignore[attr-defined]
```

Comment:
This seems to indicate that we should annotate `__init__.py` first, but I guess that's too late.